Fixes broken UI

commitsovercoffee opened the pull request:
### Updated !

Fixes #85 by adding `style="margin:0"` to misaligned headings.

* Aligns waste properties & icon under **Types of waste** on [cd workshop](https://verifa.io/work/continuous-delivery-workshop/) page.

| Before | After |
| :---------------------: | :---------------------: |
| ![waste-before](https://github.com/verifa/website/assets/129554482/5f3342a5-3754-4b0c-b88c-b3658546fefe) |![waste-after](https://github.com/verifa/website/assets/129554482/6e8e00a9-ea06-4bc9-9ebf-cb1c05d0e8a9)|

* Aligns workshop stages pointers on [cd workshop](https://verifa.io/work/continuous-delivery-workshop/) page.

| Before | After |
| :---------------------: | :---------------------: |
|![before-stage](https://github.com/verifa/website/assets/129554482/d5b3663e-255f-4c8c-91c1-c926b9c1ca38) |![after-stage](https://github.com/verifa/website/assets/129554482/1e8c36c1-a4fa-496e-a91b-634a56c2151f)|

* Aligns continuous delivery principle label and > icon on [cd workshop](https://verifa.io/work/continuous-delivery-workshop/) page.

| Before | After |
| :---------------------: | :---------------------: |
|![cd-before](https://github.com/verifa/website/assets/129554482/0776a63c-df0c-4925-b690-8adba3ec5549) |![cd-after](https://github.com/verifa/website/assets/129554482/c75ca3c1-bda7-4ce1-9f34-cdc459f45cff)|

* Fixes gap between project description and topics on [projects](https://verifa.io/projects/) page.

| Before | After |
| :---------------------: | :---------------------: |
|![projects-before](https://github.com/verifa/website/assets/129554482/204f197d-0245-4234-b390-bb2ce7aba859)|![projects-after](https://github.com/verifa/website/assets/129554482/8d1b4d67-1227-44ba-8663-b0142f1d9326)|

jlarfors commented on verifa.diff:
Assuming this was not supposed to be committed :)

commitsovercoffee replied:
oops. forgot to delete that.

jlarfors commented on src/routes/work/continuous-delivery-workshop/_waste.svelte:
```diff
@@ -62,12 +62,12 @@
 					<img class="object-contain w-12 h-12" src={waste.img} alt="waste-type" />
 					<div class="group flex items-center gap-x-4">
-						<h4 class="mb-0 group-hover:text-v-lilac">{waste.name}</h4>
+						<p class="mb-0 group-hover:text-v-lilac text-xl font-medium text-black">{waste.name}</p>
```
We'd like to keep using the `h1-4` elements to ensure consistency and avoid copying the `text-xl font-medium text-black` classes across the whole site...

The cause of all this was switching to using the tailwind [typography plugin](https://tailwindcss.com/docs/typography-plugin) across the whole site...

commitsovercoffee replied:
I had some queries / clarifications ( if you don't mind ) ...

- Heading elements  h1-h6 [should not be used to resize text](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes) hence I used `p` instead of `h4`.
- To use consistent text color, a [grayscale can be chosen](https://tailwindcss.com/docs/typography-plugin#choosing-a-gray-scale) for `prose`, and copying text-black won't be required.
- If same style is to be used at several places, Shouldn't a component be created ?

Nevertheless, I have adjusted the padding to fix misalignments.

jlarfors commented on src/lib/faqItem.svelte:
```diff
@@ -11,12 +11,12 @@
 			on:click={() => (show = !show)}
 		>
-			<h4 class="mb-0">{question}</h4>
+			<p class="mb-0 text-xl">{question}</p>
```
Same as comment further down, we want to keep using `h1-4` elements for consistency across the site.

jlarfors commented:
@commitsovercoffee thanks for the PR! That's awesome :)

I left some comments for you, would be great if you could take a look and let me know what you think.
Also if you could rebase your commits after any further changes to keep the history clean that would be much appreciated!

commitsovercoffee commented:
@jlarfors I have responded to your comments and reverted back to using h1-4 elements.

Tailwind's default typography style adds a top margin for h1-4 which is causing the misalignment. But I'm not sure how changing the default style for h1-h4 will affect other places where headings are used. So I have just adjusted the padding to fix the alignments. 🤔

jlarfors commented on verifa.diff:
You left this in the commit. Please remove it :)

jlarfors commented:
@commitsovercoffee thanks for the updated PR!

Did you have any objections to just adding `style="margin:0"` to the elements? This would remove all the margin from the headers, as added by the tailwind typography plugin, and mean we don't need to start messing around with the paddings (I'd need to test those on different screens to make sure it's responsive).

jlarfors commented:
@commitsovercoffee really sorry for the delay getting this merged. Really appreciate your contribution, thanks!
